tuf/keys.py

```diff
@@ -139,7 +149,7 @@
 
 # The crypto libraries to use in 'keys.py', set by default or by the user.
 # The following cryptography libraries are currently supported:
-# ['pycrypto', 'pynacl', 'ed25519']
+# ['pycrypto', 'pynacl', 'ed25519', 'pyca-cryptography']
```

Comment:
IIRC this comment was meant to point at a similar comment in `conf.py`, https://github.com/theupdateframework/tuf/blob/7f91f56802dd6b851ca6bc5645744b9617011d55/tuf/conf.py#L93-L95 (I probably typoed the line number, and/or they changed due to commits since.)

The point being: Both `keys.py` and `conf.py` should mention the same supported crypto libraries.